docs/cristin-proxy-swagger.yaml

```diff
@@ -520,6 +613,99 @@ paths:
             type: number
             default: '5'
           example: '10'
+        - name: institution
+          in: query
+          description: Cristin id, acronym, name, or part of the name of the coordinating institution
+          required: false
+          schema:
+            type: string
+        - name: project_manager
+          in: query
+          description: Cristin id, name or part of the name of the project manager.
+          required: false
+          schema:
+            type: string
+        - name: participant
+          in: query
+          description: Cristin id, name or part of the name of a participant of the project. 'participant' can be used more than once.
+          required: false
+          schema:
+            type: string
+        - name: keyword
+          in: query
+          description: Keyword search. '?keyword=wind energy' makes a search for projects with keywords equal to or a substring of 'wind energy'. keyword can be used more than once
+          required: false
+          schema:
+            type: string
+        - name: approval_reference_id
+          in: query
+          description: Reference id of a project approval.
+          required: false
+          schema:
+            type: string
+        - name: funding_source
+          in: query
+          description: Funding source code.
+          required: false
+          schema:
+            type: string
+        - name: sort
+          in: query
+          description: here
+          required: false
+          schema:
+            type: string
+        - name: name
+          in: query
+          description: Sorts on 'start_date' and/or 'end_date'. See Search and sort. Default sort order is on 'cristin_project_id' in ascending order.
```

Comment:
See other comment

Reply:
Name was a dummy-template which I was feeling out for each parameter.
Fixed it now

Reply:
moved description for sort in sort